src/natcap/invest/utils.py

```diff
@@ -43,6 +44,173 @@
 # leaves Projected CRS alone
 DEFAULT_OSR_AXIS_MAPPING_STRATEGY = osr.OAMS_TRADITIONAL_GIS_ORDER
 
+# the same unit registry instance should be shared across everything
+u = pint.UnitRegistry()
+u.define('dollar = [value]')  # adding a new dimension "value"
+u.define('million_dollars = 1000000 * dollar')
+u.define('currency = []')         # non-specific unit of value
+u.define('pixel = []')            # non-specific unit of area
+u.define('projection_unit = []')  # non-specific unit of length
+# used in coastal vulnerability, the DEM pixel values measure
+# elevation (length) but the specific units don't matter
+u.define('linear_unit_of_elevation = []')  # non-specific unit of length
+# add "us_survey_foot" on to the aliases because it's used in some rasters
+u.define('survey_foot = 1200 / 3937 * meter = sft = us_survey_foot')
+# Vitamin A in the crop production nutrient table is measured in IUs
+# A special unit in pharmacology that measures biologically active substances
+# May be converted to weight or volume, but conversion factors are specific
+# to the substance. I couldn't find a definition of its dimensionality.
+u.define('international_unit = [biologic_amount] = iu = IU')
+# Use u.none for unitless measurements
+u.define('none = []')
+
+AREA = {
```

Comment:
That's a good point Dave. I could also go either way on whether it still lives in `utils` or somewhere else. Maybe just an `args_spec` module...?

Reply:
In 5fefe909 and 04903b85 I moved this stuff to a new module `spec_utils.py`. I like that better than using a dictionary because with everything as top-level variables, you can use an already-defined variable in the definition of new variables. I also made all `"geometries"` values refer to constants in `spec_utils.py`. Because we're not totally sure what to do with those yet, it'll be convenient to have them all in one place in case we change them.